src/components/v5/common/Table/types.ts

```diff
@@ -29,4 +31,11 @@ export interface TableProps<T>
   renderCellWrapper?: RenderCellWrapper<T>;
   additionalPaginationButtonsContent?: React.ReactNode;
   emptyContent?: React.ReactNode;
+  getMenuProps?: (row: Row<T>) => MeatBallMenuProps | undefined;
```

Comment:
not sure if this would be beneficial, but maybe we could get rid of this and just pass a rendered meatball menu. Maybe we could get rid of both sizes too, if it's static we set the column size there, etc.
But feel free to ignore this if it would be useless